modules/builds-image-source.adoc

```diff
@@ -60,12 +64,9 @@ This feature is not supported for builds using the custom strategy.
 ====
 endif::[]
 
-/////
-[role="_additional-resources"]
-.Additional resources
+.Images on mirrored registries that require pull secrets
```

Comment:
@adambkaplan Does it make sense to add this information as a note at the end of the "Image Source" section [shown here](https://rolfedh.github.io/previews/RHDEVDOCS-2587/cicd/builds/creating-build-inputs.html#builds-image-source_creating-build-inputs)? Look for the "Images on mirrored registries that require pull secrets" subheading.

Reply:
If so, I'll remove modules/working-with-mirrored-images.adoc from this PR.

Reply:
/unassign @adambkaplan 
/assign @coreydaley

Reply:
Yes, I think that would be fine.

Reply:
Updated. Thanks.